Project doesn't support noImplicitAny

Hi, your definition file seems completely broken. When I include it in my project without even using it I get all these errors:

```
typings/main/definitions/sequelize/index.d.ts(825,15): error TS7006: Parameter 'options' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(832,13): error TS7006: Parameter 'value' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(832,20): error TS7006: Parameter 'options' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(833,12): error TS7006: Parameter 'options' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1037,19): error TS7006: Parameter 'date' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1038,15): error TS7006: Parameter 'date' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1088,11): error TS7006: Parameter 'hext' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1209,15): error TS7006: Parameter 'type' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1209,21): error TS7006: Parameter 'srid' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1229,3): error TS7010: 'construcor', which lacks return-type annotation, implicitly has an 'any' return type.
typings/main/definitions/sequelize/index.d.ts(1229,14): error TS7006: Parameter 'type' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1229,20): error TS7006: Parameter 'srid' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1230,3): error TS7010: 'construcor', which lacks return-type annotation, implicitly has an 'any' return type.
typings/main/definitions/sequelize/index.d.ts(1343,34): error TS7006: Parameter 'options' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1343,43): error TS7006: Parameter 'Model' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1346,37): error TS7006: Parameter 'options' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1346,46): error TS7006: Parameter 'Model' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1348,36): error TS7006: Parameter 'attributes' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1348,48): error TS7006: Parameter 'Model' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1350,36): error TS7006: Parameter 'dataValues' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1350,48): error TS7006: Parameter 'fields' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1350,56): error TS7006: Parameter 'Model' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1353,36): error TS7006: Parameter 'args' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1353,42): error TS7006: Parameter 'primaryKeys' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1373,39): error TS7006: Parameter 'hash' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1373,45): error TS7006: Parameter 'omitNull' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1373,55): error TS7006: Parameter 'options' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1374,25): error TS7006: Parameter 'SubClass' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1374,35): error TS7006: Parameter 'SuperClass' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1376,27): error TS7006: Parameter 'args' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1376,33): error TS7006: Parameter 'begin' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1391,15): error TS7006: Parameter 'fn' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1391,19): error TS7006: Parameter 'args' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1397,15): error TS7006: Parameter 'col' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1401,15): error TS7006: Parameter 'val' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1405,15): error TS7006: Parameter 'val' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1409,15): error TS7006: Parameter 'conditionsOrPath' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1409,33): error TS7006: Parameter 'value' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1413,15): error TS7006: Parameter 'attribute' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1413,26): error TS7006: Parameter 'comparator' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1413,38): error TS7006: Parameter 'logic' implicitly has an 'any' type.
typings/main/definitions/sequelize/index.d.ts(1418,34): error TS7006: Parameter 'obj' implicitly has an 'any' type.
```

--- a/lib/utils.d.ts
+++ b/lib/utils.d.ts
@@ -36,17 +36,17 @@ export function formatNamedParameters(sql: string, parameters: any, dialect: str
 export function cloneDeep<T>(obj: T, fn?: Function): T;
 
 /** Expand and normalize finder options */
-export function mapFinderOptions(options, Model): any;
+export function mapFinderOptions(options: any, Model: any): any;
 
 /* Used to map field names in attributes and where conditions */
-export function mapOptionFieldNames(options, Model): any;
+export function mapOptionFieldNames(options: any, Model: any): any;
 
-export function mapWhereFieldNames(attributes, Model): any;
+export function mapWhereFieldNames(attributes: any, Model: any): any;
 /** Used to map field names in values */
-export function mapValueFieldNames(dataValues, fields, Model): any;
+export function mapValueFieldNames(dataValues: any, fields: any, Model: any): any;
 
 export function isColString(value: string): boolean;
-export function argsArePrimaryKeys(args, primaryKeys): boolean;
+export function argsArePrimaryKeys(args: any, primaryKeys: any): boolean;
 export function canTreatArrayAsAnd(arr: any[]): boolean;
 export function combineTableNames(tableName1: string, tableName2: string): string;
 
@@ -66,10 +66,10 @@ export function toDefaultValue(value: any): any;
  */
 export function defaultValueSchemable(value: any): boolean;
 
-export function defaultValueSchemable(hash, omitNull, options): any;
-export function inherit(SubClass, SuperClass): any;
+export function defaultValueSchemable(hash: any, omitNull: any, options: any): any;
+export function inherit(SubClass: any, SuperClass: any): any;
 export function stack(): string;
-export function sliceArgs(args, begin): any[];
+export function sliceArgs(args: any, begin: any): any[];
 export function now(dialect: string): Date;
 export function tick(func: Function): void;
 
@@ -84,34 +84,34 @@ export function removeTicks(s: string, tickChar?: string): string;
  * Please do not use these functions directly, use Sequelize.fn and Sequelize.col instead.
  */
 export class fn {
-  constructor(fn, args);
+  constructor(fn: any, args: any);
   _isSequelizeMethod: boolean;
 
   clone(): this;
 }
 export class col {
-  constructor(col);
+  constructor(col: any);
   _isSequelizeMethod: boolean;
 }
 export class cast {
-  constructor(val, type?: string);
+  constructor(val: any, type?: string);
   _isSequelizeMethod: boolean;
 }
 export class literal {
-  constructor(val);
+  constructor(val: any);
   _isSequelizeMethod: boolean;
 }
 export class json {
-  constructor(conditionsOrPath, value);
+  constructor(conditionsOrPath: any, value: any);
   _isSequelizeMethod: boolean;
 }
 export class where {
-  constructor(attribute, comparator, logic?);
+  constructor(attribute: any, comparator: any, logic?: any);
   _isSequelizeMethod: boolean;
 }
 
 export const validateParameter: typeof parameterValidator;
-export function formatReferences(obj): any;
+export function formatReferences(obj: any): any;
 
 import Promise = require('./promise');
 export {Promise};

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "noImplicitAny": true,
     "module": "commonjs",
     "moduleResolution": "node"
   },

